src/main/java/noppes/npcs/entity/EntityProjectile.java

```diff
@@ -138,6 +139,7 @@ public EntityProjectile(World par1World, EntityLivingBase par2EntityLiving, Item
         if (isNPC) {
         	this.npc = (EntityNPCInterface) this.thrower;
         	this.getStatProperties(this.npc.stats);
+        	this.destroyTerrain = !this.npc.projectilesKeepTerrain;
```

Comment:
Change to:

`this.destroyTerrain = !this.npc.stats.projectilesKeepTerrain;`

After moving 'projectilesKeepTerrain' into 'DataStats'.